release-notes/release-2.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: February 13, 2024
+---
+
+# Features
+- PHP Azure Build Pipelines
+- [RUM Mobile - React Native](https://docs.middleware.io/rum/rum-react)
+- Spend Analysis charts
+- Trace correlation drill down from specific log messages
+- Price calculator that analyzes your bill based on the usage of different products
+- Sort and group attributes of log-collected data
```

Comment:
Ability to sort and group by any attribute on the logs page

(This may warrant a quick gif)

Reply:
I think it is fine to include in the release notes but there is a small UX bug where the attribute overlaps the filter button. Dont think we should include a GIF. See here for ticket: https://linear.app/middleware/issue/ENG-1907/logs-sort-by-attribute